app/schema/api.py

```diff
@@ -243,6 +245,29 @@ class AddPriceOut(AddPriceIn):
     id: int
 
 
+class AddDiscountIn(BaseModel):
+    """AddDiscountIn."""
+
+    vlab_id: UUID
+    discount: Annotated[Decimal, Field(ge=D0, le=D1)]
```

Comment:
For clarification, is it acceptable to create a discount with value 0 or 1? My thoughts:
- 0 would be equal to no-discount, so it can be used to invalidate an existing discount since it cannot be updated, for example
- 1 would be equal to make the service free

If these are the intended use cases, we could add a comment somewhere in the code or docs.
We could also test the edge cases explicitly.

Reply:
Now there is a note under the API docstring, as well as tests for edge cases: for discount 0 and 1.